### PR TITLE
OSSMDOC-525: preliminary Jupiter updates for dist tracing and jaeger files.

### PR DESCRIPTION
**Pull request by JStickler**
Version(s): 4.6 - 4.11

Issue: [OSSMDOC-525](https://issues.redhat.com/browse/OSSMDOC-525)

Link to docs preview: No visible changes in output?

This PR is not intended to address all required Jupiter changes, it only includes some preliminary cleanup done while scoping Jupiter work for the Epic [OSSMDOC-523](https://issues.redhat.com/browse/OSSMDOC-523).  This PR may include:

- updating IDs to match headings
- Deleting comments or notes
- replacing text with attributes
- Adding empty lines required by formatting
- Deleting double spaces after a period

Peer review: maxwelldb

### CODE REVIEW

File: modules/distr-tracing-architecture.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ This module included in the following assemblies:
 -dist_tracing_arch/distr-tracing-architecture.adoc
 ////
 :_content-type: CONCEPT
-[id="distr-tracing-architecture_{context}"]
+[id="distributed-tracing-architecture_{context}"]
```
Comment (maxwelldb) [inline modules/distr-tracing-architecture.adoc:7]: According to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata, module IDs must match file names. 

Check other modules for the same issue.

**Review by maxwelldb — COMMENTED** [review]
LGTM aside from one note.

File: modules/distr-tracing-architecture.adoc (hunk 1 above)
Comment (JStickler) [reply modules/distr-tracing-architecture.adoc:7]: Argh, missed one!

**Pull request merged**